config/default/pyproject.toml.j2

```diff
@@ -1,3 +1,6 @@
+[build-system]
+requires = ["setuptools>=68.2", "wheel"]
```

Comment:
`setuptools>=68.2` should already tell the frontend to install `wheel` via the `get_requires_for_*` build hook.

Reply:
Aha.  Let me check.  That happens on [these lines](https://github.com/pypa/setuptools/blob/2384d915088b960999ca74fb81ce70bffd17b082/setuptools/build_meta.py#L354-L358).  This seems to arrange that when you want to build a wheel, the `wheel` package gets installed, but not when you only want to build an `sdist`.  Seems useful.  And `git blame` says this code has been there for at least this entire year, so we can indeed rely on it.

Thanks!  I will update this.